src/main/java/frc/robot/Constants.java

```diff
@@ -15,12 +15,28 @@
 public final class Constants {
     public static final double stickDeadband = 0.1;
 
+
+    /* PID Rotation */ 
+    public static final double ROTATE_KP = 0.02; //0.0222
+    public static final double ROTATE_KI = 0.0;
+    public static final double ROTATE_KD = 0.001;
+    public static final double ROTATE_VELOCITY = 200.0;
+    public static final double ROTATE_ACCELERATION = 400.0;
+
+    /* slow mode */
+    public static final double SLOW_MODE_PERCENT_TRANSLATION = 0.5;
+    public static final double SLOW_MODE_PERCENT_STRAFE = 0.5;
+    public static final double SLOW_MODE_PERCENT_ROTATION = 0.5;
+
+    public static final double AUTO_ROTATE_DEADBAND = 0.0;
+
     public static final class Swerve {
 
-        public static final double maxEncoderVoltage = 5;
-        public static final int pigeonID = 1;
+        public static final double maxEncoderVoltage = 5.057;
```

Comment:
does this name make sense? We only use Analog Input encoders on swerve but what if we use them elsewhere?